frontends/web/src/routes/settings/manage-accounts.tsx

```diff
@@ -41,13 +44,17 @@ type TShowTokens = {
 }
 
 interface State {
+    editAccountCode?: string;
+    editAccountNewName?: string;
+    editErrorMessage?: string;
```

Comment:
just an idea, not sure: maybe it's easier to put those into one object:

`editState?: { accountCode: string; ... }`

Reply:
setState doesn't deep merge